package.json

```diff
@@ -152,7 +152,7 @@
     "ts-node": "^10.9.1",
     "typed-css-modules": "^0.7.0",
     "typed-css-modules-loader": "0.0.18",
-    "typescript": "^4.9.5",
+    "typescript": "5.3.3",
```

Comment:
I've updated to include the caret. I want to be consistent! :)

That's an old habit of mine (to take them out), one of my prior jobs went so far as to have eslint rules to prevent anything but direct version usages and ran a CI check to catch strays.

Not every package follows semantic versioning properly, and every now and then we'd find an issue while trying to roll out an important change because the CI/CD would pick up a newer package than what we had locally and it would clog up the pipeline while we fixed it.

Things seem to be better these days, especially if you pick well maintained libraries to build from (typescript would be one of them).

Thanks Jacob.